src/tests/functional/shared_test_classes/include/shared_test_classes/base/benchmark.hpp

```diff
@@ -0,0 +1,193 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <cctype>
+#include <iostream>
+
+#include "layer_test_utils.hpp"
+#include "ov_subgraph.hpp"
+
+namespace LayerTestsDefinitions {
+
+template <typename BaseLayerTest>
+class BenchmarkLayerTest : public BaseLayerTest {
+    static_assert(std::is_base_of<LayerTestsUtils::LayerTestsCommon, BaseLayerTest>::value,
+                  "BaseLayerTest should inherit from LayerTestsUtils::LayerTestsCommon");
+
+public:
+    static constexpr int kDefaultNumberOfAttempts = 100;
+
+    void RunBenchmark(const std::initializer_list<std::string>& node_type_names,
+                      const std::chrono::milliseconds warmupTime = std::chrono::milliseconds(2000),
+                      const int numAttempts = kDefaultNumberOfAttempts) {
```

Comment:
But it will be exposed to all file where this benchmark.hpp will be included ...
Do we really want this ?

Reply:
you use the different code style for 2 arguments (`warmupTime` & `numAttempts`). It looks not so good. I meant to use the same mechanism, because the constants are used twice. Looks that my proposal is logical. @ilyachur do you have any suggestions to solve the discussion?

Reply:
Fixed